starter-code/movies/views/celebs/index.ejs

```diff
@@ -0,0 +1,19 @@
+<h2>Celebrities List</h2>
+
+<ul>
+	<% celebs.forEach(el => {%>
+<li>
+	<a href="/celebs/<%= el._id%>/show"> <%= el.name%></a> 
+	<form action="/celebs/<%= el._id%>/delete" method="post" accept-charset="utf-8">		
+	<button type="submit">REMOVE</button><br>
+	</form>
+	<form action="/celebs/<%= el._id%>/edit" method="get" accept-charset="utf-8">		
+	<button type="submit">EDIT</button><br>
+	</form>
+	Occupation: <%=el.occupation%><br>
+	"<%= el.catchPhrase%>"<br>
+	<%= el._id%>
```

Comment:
I don't think you should render the id in the view. The user doesn't care about this meaningless number, which is only useful for databases.

Reply:
Yeah haha, I used it for debug purposes and completely forgot to remove it... My bad